### PR TITLE
Bugfix issue #1274

### PR DESCRIPTION
**Pull request by rob-nn**
## Change Description

Changing the method redact from class ImageAnalyzerEngine to enable select others languages in addition "en".
Now is possible use other languages like "pt" and configure tesseract to use the wished language

## Issue reference

This PR fixes issue #1274

## Checklist

- [ x ] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [ x ] I have signed the CLA (if required)
- [  x ] My code includes unit tests
- [ x ] All unit tests and lint checks pass locally
- [ x ] My PR contains documentation updates / additions if required

### CODE REVIEW

**Comment by rob-nn** [comment]
@microsoft-github-policy-service agree

File: presidio-image-redactor/tests/test_image_analyzer_engine.py [hunk 1]
```diff
@@ -385,4 +385,14 @@ def compare_color(actual_pixels, expected_color, threshold=10):
...
 
+def test_check_analyze_supports_language_param(get_mock_png):
+    """Test that the method analyze from class ImageAnalyzerEngine
+    supports the language parameter. 
+    Before there where a bug: 
+        "TypeError: presidio_analyzer.analyzer_engine.AnalyzerEngine.analyze() got multiple values for keyword argument 'language'"
+        
+    :param get_mock_png: The mock PNG image
+    """
+    redacted = ImageAnalyzerEngine().analyze(get_mock_png, language='en')
```
Comment (omri374) [inline presidio-image-redactor/tests/test_image_analyzer_engine.py:396]: Can you please add another test where the language isn't 'en' and the results are empty?

Comment (rob-nn) [reply presidio-image-redactor/tests/test_image_analyzer_engine.py:396]: Ok. I will do

Comment (rob-nn) [reply presidio-image-redactor/tests/test_image_analyzer_engine.py:397]: Done.
